whisper/whisperv6/message.go

```diff
@@ -353,7 +342,7 @@ func (msg *ReceivedMessage) SigToPubKey() *ecdsa.PublicKey {
 	return pub
 }
 
-// hash calculates the SHA3 checksum of the message flags, payload and padding.
+// hash calculates the SHA3 checksum of the message flags, auxiliary field, payload and padding.
```

Comment:
i insist on auxiliary field.
please see my corresponding comment above.

Reply:
and I insist on not using it, because it's not descriptive. Could you at least provide a reason?

Reply:
sorry, i did not realize that that comment above was hidden as outdated. i copy it here as well:

i have chosen this name (AuxiliaryField) instead of PayloadSizeField, because if I need to reference its size, it would become really incomprehensible: PayloadSizeFieldSize? SizeOfPayloadSize? Both are very bad. So, for the sake of consitency, it is now called AuxiliaryField everywhere.

Reply:
both names are long and way better than Auxiliary. As you are aware, auxiliary means that it sits on the side to help. That could very well be the number of bananas than the user ate before they sent the message, it would still be an auxiliary field. `PalyloadLengthFieldSize` is explicit: it says what the field contains: the length of the payload, not that it sits on the size and is somehow "here to help".

Reply:
indeed